packages/guideline-blocks-settings/src/components/Attachments/Attachments.tsx

```diff
@@ -184,6 +190,12 @@ export const Attachments = ({
                                     <DragOverlay>
                                         {draggedItem && (
                                             <AttachmentItem
+                                                download={() =>
```

Comment:
I think for the drag overlay we might not need this handler. What do you think? To avoid accidental downloads when user just wants to reorder.

Reply:
You are right, improved it.